_gettingstarted/highcharts-core.md

```diff
@@ -0,0 +1,249 @@
+    'series': [{
+        'name': 'Series 1',
+        'data': [10, 20, 15, 25, 30]
+    }, {
+        'name': 'Series 2',
+        'data': [5, 15, 10, 20, 25]
+    }]
+}
+advanced_bar_chart.set_dict_options(advanced_bar_options)
+advanced_bar_chart.save_file('advanced_bar_chart.html')
+```
+![Bar Chart](./../assets/images/getting-started/highcharts/bar-chart.png)
+
+```python
+# Column Chart
+advanced_column_chart = highcharts.Highchart()
+
+advanced_column_options = {
+    'chart': {
+        'type': 'column'
+    },
+    'title': {
+        'text': 'Advanced Highcharts Column Chart'
+    },
+    'xAxis': {
+        'categories': ['Jan', 'Feb', 'Mar', 'Apr', 'May']
+    },
+    'yAxis': {
+        'title': {
+            'text': 'Values'
+        }
+    },
+    'series': [{
+        'name': 'Series 1',
+        'data': [10, 20, 15, 25, 30]
+    }, {
+        'name': 'Series 2',
+        'data': [5, 15, 10, 20, 25]
+    }]
+}
+advanced_column_chart.set_dict_options(advanced_column_options)
+advanced_column_chart.save_file('advanced_column_chart.html')
+```
+![Column Chart](./../assets/images/getting-started/highcharts/column-chart.png)
+
+```python
+# Column Stack Chart
+advanced_column_stack_chart = highcharts.Highchart()
+
+advanced_column_stack_options = {
+    'chart': {
+        'type': 'column'
+    },
+    'title': {
+        'text': 'Advanced Highcharts Column Stack Chart'
+    },
+    'xAxis': {
+        'categories': ['Jan', 'Feb', 'Mar', 'Apr', 'May']
+    },
+    'yAxis': {
+        'title': {
+            'text': 'Values'
+        }
+    },
+    'series': [{
+        'name': 'Series 1',
+        'data': [10, 20, 15, 25, 30]
+    }, {
+        'name': 'Series 2',
+        'data': [5, 15, 10, 20, 25]
+    }]
+}
+advanced_column_stack_chart.set_dict_options(advanced_column_stack_options)
+advanced_column_stack_chart.save_file('advanced_column_stack_chart.html')
+```
+![Column Stack Chart](./../assets/images/getting-started/highcharts/column-stack-chart.png)
+
+
+## Options
+`chart` > `type` : Specifies the type of the chart eg: line, bar, area, column etc.,
+`text`: Specifies the title of the chart.  
+`xAxis` > `categories`: Defines the categories (labels) for the X-axis. In this example, the X-axis will have categories 'Jan', 'Feb', 'Mar', 'Apr', and 'May'.  
+`yAxis` > `title`: Specifies the title of the Y-axis.  
+`series`:This is an array of series objects. Each series object represents a set of data to be plotted on the chart.  
+`name`: The name of the series.  
+`data`: An array of data points for the series.  
+`plotOptions`: Allows you to set options that are common to all plot options (series types).In this example, it is used to configure the column chart options.  
+`column`: Specifies that the configuration is for the column chart type.  
+`stacking`: Sets the stacking option for the column chart. In this case, it's set to 'normal', meaning the columns are stacked on top of each other.  
+
+## We have Extended Libraries 
```

Comment:
Instead of We have, you can just start the heading as Extended Libraries or Useful Links, etc.
Adding "We have", makes it look like the article is from highcharts and not pip trends,